setup-scripts/install.sh

```diff
@@ -94,7 +94,20 @@ case $i in
 esac
 done
 
-msg "You're about to install Sparkswap. Good for you!" $GREEN
+echo ""
```

Comment:
by doing this, the logo will show up twice (cause we'll run the build spec). Is that OK?

Reply:
Actually, will show up 3 times for the current install

Reply:
You can ignore this. Its behind a build flag